### ISSUE TITLE
CSS issues in multiple editor parts

### ISSUE THREAD

**Issue by fernandomm** [Automattic/isolated-block-editor#160]
I was running this project locally and noticed some CSS issues that aren't present in the "original" Gutenberg editor.

Some examples:

1.  When a column is focused, the border is off

![Captura de Tela 2022-06-16 às 10 52 12](https://user-images.githubusercontent.com/439774/174085434-f109f879-c75e-47ac-8a2b-6dcf7ec9113e.png)

2. The close icon border has problems

![Captura de Tela 2022-06-16 às 10 31 59](https://user-images.githubusercontent.com/439774/174086314-a81501fe-e4fc-4226-abd2-eeea86e8bc95.png)

3. When searching for a new block, horizontal scrollbars are created when cursor is hover an item

![Captura de Tela 2022-06-16 às 10 50 37](https://user-images.githubusercontent.com/439774/174086077-6f6edb16-9952-4110-8db9-ca30528259d1.png)

4. The inspector CSS seems to be missing

![Captura de Tela 2022-06-16 às 10 58 27](https://user-images.githubusercontent.com/439774/174086824-8ff31673-4249-43e4-9d9a-4f87c60b0300.png)

At least the 3 first issues seems to be caused by this CSS https://github.com/Automattic/isolated-block-editor/blob/9451d68a4e002a39f9badbb6fdef4f247325a4c8/src/components/block-editor/style.scss#L40-L50

After removing it the 3 first issues were fixed without any noticeable side effects. Is it safe to remove these custom rules? I can submit a PR if you're ok with it.

**Comment by johngodley** [comment]
Sure, a PR would be great! It's likely this is old CSS that was relevant to a previous version of Gutenberg and may now be causing problems.

I was able to reproduce 1.

For 2 this is seen in core Gutenberg too. The icon shouldn't actually appear, and this was fixed in #157

I can't reproduce 3, but maybe there are some other local differences.

4 is probably worth splitting off into a separate issue.
